src/databases/CGNS/CGNSCommonPluginInfo.C

```diff
@@ -64,6 +67,75 @@ CGNSCommonPluginInfo::SetupDatabase(const char *const *list,
 {
     avtDatabase *db = NULL;
 
+    //
```

Comment:
Does everything still work with a `.visit` file too? Only reason I ask is that it looks like the release notes update no longer mentions the `.visit` file approach which suggests you've effectively removed support for that and replaced it with this? If so, I think its best to also keep the `.visit` file functionality so that other parallel (file-per-processor) CGNS data producers who may use different file naming conventions can still use a `.visit` file to work.